From b19d9650d18f347b31585d809f88c3bad795839d Mon Sep 17 00:00:00 2001
From: Theo GOUREAU <theo.goureau-ext@st.com>
Date: Mon, 21 Oct 2024 10:10:53 +0200
Subject: [PATCH 1335/1695] dt-bindings: soc: remove LTDC rotation from MP21
 RISUP list

Remove LTDC rotation from MP21 RISUP list.

Upstream-Status: Inappropriate [ST downstream https://github.com/STMicroelectronics/optee_os.git e705fbbc2c66cd6362cccdb0cc845435f599c1af dt-bindings: soc: remove LTDC rotation from MP21 RISUP list]
Change-Id: I4986689837846a87981708d85b04e8cf5f375622
Signed-off-by: Theo GOUREAU <theo.goureau-ext@st.com>
Reviewed-on: https://gerrit.st.com/c/mpu/oe/optee/optee_os/+/412804
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
ACI: CIBUILD <MDG-smet-aci-builds@list.st.com>
Reviewed-by: Thomas BOURGOIN <thomas.bourgoin@foss.st.com>
Domain-Review: Yann GAUTIER <yann.gautier@foss.st.com>
---
 core/include/dt-bindings/soc/stm32mp21-rifsc.h | 1 -
 1 file changed, 1 deletion(-)

diff --git a/core/include/dt-bindings/soc/stm32mp21-rifsc.h b/core/include/dt-bindings/soc/stm32mp21-rifsc.h
index f37c1aa63..d7f394eea 100644
--- a/core/include/dt-bindings/soc/stm32mp21-rifsc.h
+++ b/core/include/dt-bindings/soc/stm32mp21-rifsc.h
@@ -93,7 +93,6 @@
 #define STM32MP21_RIFSC_ICACHE_DCACHE_ID	118
 #define STM32MP21_RIFSC_LTDC_L0L1_ID		119
 #define STM32MP21_RIFSC_LTDC_L2_ID		120
-#define STM32MP21_RIFSC_LTDC_ROT_ID		121
 #define STM32MP21_RIFSC_OTFDEC1_ID		125
 #define STM32MP21_RIFSC_IAC_ID			127
 
-- 
2.47.2

